test/e2e/toolchaincluster_test.go

```diff
@@ -126,13 +126,22 @@ func verifyToolchainCluster(t *testing.T, await *wait.Awaitility, otherAwait *wa
 		// wait for toolchaincontroller to reconcile
 		time.Sleep(1 * time.Second)
 
-		// then the ToolchainCluster should be offline
-		require.NoError(t, err)
-		_, err = await.WaitForToolchainCluster(t,
-			wait.UntilToolchainClusterHasName(toolchainCluster.Name),
-			wait.UntilToolchainClusterHasCondition(toolchainv1alpha1.ToolchainClusterOffline),
-		)
+		// then the ToolchainCluster should be Not Ready
 		require.NoError(t, err)
+		if hostoperator {
+			_, err = await.WaitForToolchainCluster(t,
+				wait.UntilToolchainClusterHasName(toolchainCluster.Name),
+				wait.UntilToolchainClusterHasConditionAndFalseStatus(toolchainv1alpha1.ConditionReady),
```

Comment:
can you also verify that the reason is set to the expected value as well please?

Reply:
okay updating it now

Reply:
updated in [7af0bc7](https://github.com/codeready-toolchain/toolchain-e2e/pull/1005/commits/7af0bc7861f76b1ecefc23af4bf20a9cf6319d25)